presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -474,7 +474,7 @@ protected String toSqlType(Type type)
             if (varcharType.isUnbounded()) {
```

Comment:
typo in the cmt msg

Reply:
what's wrong?

Reply:
hm.... Have you fixed something? I thought I saw something like `Lenght`

Reply:
hm... i don't recall.